### DISCUSSION TITLE
open-Webui Listen on localhost flexibility - Proxy

### DISCUSSION THREAD

**Discussion by wixyskywriter** [open-webui/open-webui#7385] [Ideas]
In some cases open-webui is required to listen only on localhost. Nginx proxy will be used to serve https for outside network. seems now it is hardcoded in __init__.py:    host: str = "0.0.0.0", it would be perfect to make this dynamic and use an environment variable. change the value in the init file will get overridden the next update.

**Comment by wsargent** [comment]
`/usr/local/bin/uvx --python 3.11 open-webui@latest serve --host 127.0.0.1 --port 3000`
